source/json/highlights.schema.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {},
+  "id": "http://example.com/example.json",
+  "items": {
+    "id": "/items",
+    "properties": {
+      "description": {
+        "id": "/items/properties/description",
+        "type": "string"
+      },
+      "footer": {
+        "id": "/items/properties/footer",
+        "type": [
+          "string",
```

Comment:
See if there's a way to indicate this is going to be a string that's rendered as HTML. If so, adjust it. If not that's fine too, just not ideal for clarity.